include/llbuild/Basic/Tracing.h

```diff
@@ -15,6 +15,30 @@
 
 #include "llvm/ADT/StringRef.h"
 
+#if __has_include("TracingMacros.h")
```

Comment:
These seems a little convoluted, but I agree it might make it easier to compose custom tracing endpoints.

Can you at least include a comment here about why this is phrased the way this is (basically, so that vendored llbuild installations can provide custom tracing entry points).

Reply:
Done!